src/rest.cpp

```diff
@@ -73,18 +75,18 @@ static bool RESTERR(HTTPRequest* req, enum HTTPStatusCode status, std::string me
  *                  context is not found.
  * @returns         Pointer to the node context or nullptr if not found.
  */
-static NodeContext* GetNodeContext(const util::Ref& context, HTTPRequest* req)
+static NodeContext* GetNodeContext(const std::any& context, HTTPRequest* req)
 {
-    NodeContext* node = context.Has<NodeContext>() ? &context.Get<NodeContext>() : nullptr;
-    if (!node) {
+    NodeContext* node_context = util::AnyPtr<NodeContext>(context);
```

Comment:
nit: Could be more concise without losing much details:
```suggestion
    auto node_context = util::AnyPtr<NodeContext>(context);
```

(Here and in other places when `util::AnyPtr<T>` is used)

Reply:
Good idea. I agree that for the reader it is obvious enough to see that "AnyPtr<T>(...)" returns type "T*" and using "auto" makes sense here. Done for all instances.